usaspending_api/settings.py

```diff
@@ -313,6 +313,7 @@ def _configure_database_connection(environment_variable, test_options=None):
     config = dj_database_url.parse(os.environ.get(environment_variable), conn_max_age=CONNECTION_MAX_SECONDS)
     config["OPTIONS"] = {**config.setdefault("OPTIONS", {}), **default_options}
     config["TEST"] = {**test_options, "SERIALIZE": False}
+    config["DISABLE_SERVER_SIDE_CURSORS"] = True
```

Comment:
Why did you make this change? It can have serious performance implications.

Reply:
The Disable Server Side Cursor needed to be set to True to fix the DJango Cursor errors we were seeing after updating Django to the newest version. Local testing shows no differences in performance, however I have only tested a handful of apis, but I am not sure if testing in local environment is enough to test performance.